guides/README.md

```diff
@@ -219,19 +232,22 @@ See the [reference template](https://raw.githubusercontent.com/redhat-documentat
 * [`snip`](https://redhat-documentation.github.io/modular-docs/#using_text_snippets_or_text_fragments_writing-mod-docs): Files starting with `snip_` contain snippets that are reused throughout multiple guides, e.g. admonitions.
 Snippets do not require an ID.
 
-### Images
-
-Each guide must have an `images/` subdirectory with `images/common` symlink into the `common/images/` directory.
-Images local to the guide shall be kept in the `images/` directory.
-Images which are supposed to be reused across guides shall be kept in the `images/common/` directory.
-Subdirectories can be created and are actually recommended.
+### Creating new guides
 
-To insert an image, use `image::common/global_image.png` or `image::local_image.png`.
+Each guide must be in a separate directory in guides/ prefixed with "doc-". The following requirements must be met:
 
-You should create upstream diagrams using [diagrams.net](https://www.diagrams.net/).
-Place the editable diagram in `drawio` format in `guides/image-sources/`.
-For inclusion in the content, export diagrams to SVG and place them as described above.
+* Top-level file `master.adoc`.
+* Directory `topics` with actual content.
```

Comment:
```suggestion
```

This is no longer a requirement. For example, all content for "Managing Hosts" is keep within "guides/common/modules/".